tests/integration/src/test/resources/org/mule/test/el/expression-language-config.xml

```diff
@@ -24,7 +24,7 @@
 
     <flow name="flow">
         <expression-component>regexWrap('hello')</expression-component>
-        <test:assert expression="#[payload == 'foo']" />
+        <validation:is-true expression="#[payload == 'foo']"/>
```

Comment:
In order to use validation extension, no other place or test is using it (from a config at least). The reason was to have an integration test that uses the validation extension on isolated test mode.